packages/mirror-media-next/components/index/forum-headlines-preview.js

```diff
@@ -0,0 +1,200 @@
+  }
+
+  ${({ theme }) => theme.breakpoint.xl} {
+    flex-direction: row;
+    width: auto;
+  }
+`
+
+const Li = styled.li`
+  position: relative;
+  padding-left: 24px;
+  min-height: 84px;
+  display: flex;
+  gap: 8px;
+  flex-direction: column;
+  justify-content: space-between;
+  line-height: 1.5;
+
+  ${({ theme }) => theme.breakpoint.md} {
+    min-height: 56px;
+  }
+
+  ${({ theme }) => theme.breakpoint.xl} {
+    min-height: 84px;
+    gap: 16px;
+    flex: 1 1 0;
+  }
+
+  &::before {
+    content: '';
+    display: block;
+    position: absolute;
+    left: 0;
+    width: 8px;
+    height: 100%;
+    background: ${({ theme }) => theme.color.brandColor.lightBlue};
+  }
+`
+
+const Headline = styled.h5`
+  font-weight: 500;
+  font-size: 18px;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  &:hover {
+    text-decoration: underline;
+  }
+`
+
+const Timestamp = styled.p`
+  color: #bcbcbc;
+  font-size: 14px;
+`
+/**
+ * Check if a given date string is valid
+ * @param {string} str
+ * @returns {boolean}
+ */
+function isValidDateString(str) {
+  return !isNaN(new Date(str).getTime())
+}
+
+/**
+ * Compare updatedAt and publishedDate to decide display time
+ * @param {string} updatedAt
+ * @param {string} publishedDate
+ * @returns {string}
+ */
+function getDisplayTime(publishedDate, updatedAt) {
+  const isPublishedValid = isValidDateString(publishedDate)
+  const isUpdatedValid = isValidDateString(updatedAt)
+
+  if (isPublishedValid && isUpdatedValid) {
+    return new Date(updatedAt).getTime() > new Date(publishedDate).getTime()
+      ? updatedAt
+      : publishedDate
+  }
+
+  if (isUpdatedValid) return updatedAt
+  if (isPublishedValid) return publishedDate
+
+  return ''
+}
+
+/**
+ * Convert a UTC timestamp string to Taiwan local date and time strings
+ * @param {string} utcString - UTC timestamp
+ * @returns {{ date: string, time: string }} Object containing date and time strings
+ */
+function formatUtcToDateTime(utcString) {
+  const dateObj = new Date(utcString)
+
+  const date = dateObj.toLocaleDateString('zh-TW', {
+    timeZone: 'Asia/Taipei',
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  })
+
+  const time = dateObj.toLocaleTimeString('zh-TW', {
+    timeZone: 'Asia/Taipei',
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+    hour12: false,
+  })
+
+  return { date, time }
+}
+
+/**
+ * @typedef {Object} ExternalHeadline
+ * @property {string} id
+ * @property {string} title
+ * @property {string} slug
+ * @property {string} updatedAt - UTC timestamp when the headline was last updated
+ * @property {string} publishedDate - UTC timestamp when the headline was published
+ */
+
+/**
+ * @param {Object} props
+ * @param {ExternalHeadline[]} props.forumHeadlines - Array of latest forum headlines
+ */
+
+export default function ForumHeadlinesPreview({ forumHeadlines }) {
```

Comment:
[nit]
為了防止之後要用的人忘記傳值進來，可以加上預設值防呆。
```suggestion
export default function ForumHeadlinesPreview({ forumHeadlines = [] }) {
```

Reply:
已更新，連同上一個任務 PromoVideoList 相關區塊也更新。